protocol_structs.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I'd suggest toml just because it would be less structure fluff:

```toml
EpochInfoV4 = 5000924478162373578
EpochInfo = 10584269952899582257
```

making it easier to eyeball differences in. Sorting it also needn't care about the last comma which is significant in JSON.